sources/advertisements/VRM New Core/Controllers/VRMSelectFinalResultController.swift

```diff
@@ -13,25 +13,37 @@ final class VRMSelectFinalResultController {
         self.dispatch = dispatch
     }
     
-    func process(with state: PlayerCore.State) {
+    func process(with state: PlayerCore.State, isFailoverEnabled: Bool) {
```

Comment:
You can pass `isFailoverEnabled` value in `init`

Reply:
Done